### PR TITLE
feat: new cve, woop

### PR DESCRIPTION
**Pull request by janl**

### CODE REVIEW

File: src/cve/2020-1955.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+:Vendor: The Apache Software Foundation
+
+Description
+===========
+
+CouchDB version 3.0.0 shipped with a new configuration setting that
+governs access control to the entire database server called
+`require_valid_user_except_for_up`. It was meant as an extension to the
+long standing setting `require_valid_user`, which in turn requires that
```
Comment (flimzy) [inline src/cve/2020-1955.rst:33]:
```suggestion
long-standing setting `require_valid_user`, which in turn requires that
```

File: src/cve/2020-1955.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+However, the implementation of this made an error that lead to not
+enforcing credentials on any endpoint, when enabled.
+
+CouchDB versions :ref:`3.0.1 <release/3.0.1>` and :ref:`3.1.0
+<release/3.1.0>` fix this issue.
+
+Mitigation
+==========
+
+Users that have not enabled `require_valid_user_except_for_up` are not
```
Comment (flimzy) [inline src/cve/2020-1955.rst:50]: Minor nit; "that" is not incorrect, but "who" is probably more common, and less "objectifying", if that's a thing.

```suggestion
Users who have not enabled `require_valid_user_except_for_up` are not
```

File: src/cve/2020-1955.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+CouchDB versions :ref:`3.0.1 <release/3.0.1>` and :ref:`3.1.0
+<release/3.1.0>` fix this issue.
+
+Mitigation
+==========
+
+Users who have not enabled `require_valid_user_except_for_up` are not
+affected.
+
+Users that have it enabled can either disable it again, or upgrade to
```
Comment (janl) [inline src/cve/2020-1955.rst:53]:
```suggestion
Users who have it enabled can either disable it again, or upgrade to
```

**Comment by janl** [comment]
@flimzy ta!

File: src/cve/2020-1955.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _cve/2020-1955:
+
+===========================================================
+CVE-2020-1955: Apache CouchDB Remote Privilege Escalations
```
Comment (janl) [inline src/cve/2020-1955.rst:16]:
```suggestion
CVE-2020-1955: Apache CouchDB Remote Privilege Escalation
```

**Pull request merged**